src/components/OfficeSections/OfficeSectionsComponent.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Header,
+  Table,
+  Pagination,
+  Segment,
+  Divider,
+  Button
+} from 'semantic-ui-react';
+import _ from 'lodash';
```

Comment:
Helps to identify which functions from lodash you are using. This helps webpack tree-shaking to exclude what you don't you. Considering lodash is big, this is definitely something you should do